plugins/statistics.koplugin/calendarview.lua

```diff
@@ -443,14 +443,19 @@ function CalendarView:init()
         }
     end
 
-    local outer_padding = Size.padding.large
+    self.outer_padding = Size.padding.large
     self.inner_padding = Size.padding.small
 
     -- 7 days in a week
-    self.day_width = math.floor((self.dimen.w - 2*outer_padding - 6*self.inner_padding) / 7)
+    self.day_width = math.floor((self.dimen.w - 2*self.outer_padding - 6*self.inner_padding) / 7)
     -- Put back the possible 7px lost in rounding into outer_padding
-    outer_padding = math.floor((self.dimen.w - 7*self.day_width - 6*self.inner_padding) / 2)
-    self.content_width = self.dimen.w - 2*outer_padding
+    self.outer_padding = math.floor((self.dimen.w - 7*self.day_width - 6*self.inner_padding) / 2)
+
+    self.inner_dimen = Geom:new{
+        w = self.dimen.w - 2*self.outer_padding,
+        h = self.dimen.h - self.outer_padding, -- no bottom padding
+    }
```

Comment:
Isn't there some mess around here, using self.outer_padding (computing from width stuff) for self.inner_dimen.h ?

Reply:
Yeah, self.outer_padding gets corrected based on something computed from an estimate of itself first.

Reply:
I'm not sure how necessary it is (and how much the estimate and the final result changes, but as it's only done once, I figured I'd let it be ;p).